### DISCUSSION TITLE
How to Refresh all Queries in Dashboard from a Custom Plugin

### DISCUSSION THREAD

**Discussion by jayaraj** [grafana/grafana#55784] [Dashboards, Panels & Visualizations]
How do we Refresh all Queries in Dashboard from a Custom Plugin ?

I tried eventBus.publish(new RefreshEvent()); Which is not working.

Thanks & regards,

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
